### PR TITLE
Java Quick Starter

### PR DESCRIPTION
**Pull request by Basfar**
### What change does this PR introduce?

I created a Java Quick Starter file for the newly create  nove-java-sdk.

The implementation done was similar to that of Kotlin, as you provide sample implementation to trigger notification, events, topics etc.

### Why was this change needed?
It was new implementation

### CODE REVIEW

**Comment by LetItRock** [comment]
@unicodeveloper fyi

File: docs/docs/overview/quickstart/get-started-with-java.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+- [Send Notifications via Topics.](#topics)
+
+Let's get started! :muscle:
+
+## Requirements
+
+To be able to use this Quickstart, you would need to have the following:
+
+- A Novu account, if you don't have one yet, [sign up](https://web.novu.co) for free.
+- A working Java development environment with JDK 17 and Springboot Framework of 4.0.0+.
```
Comment (mayorJAY) [inline docs/docs/overview/quickstart/get-started-with-java.md:26]: The Spring version I can see in the quickstart code is v3.1.2 but you mentioned v4.0 here. I think you should reconcile this

File: docs/docs/overview/quickstart/get-started-with-java.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+Sync your project, and you should have the artifacts downloaded.
+
+To use the SDK, you need to have your API key handy, it can be gotten from [settings page](https://web.novu.co/settings) of the web portal.
+
+To initialize the library, you can create an instance of `Novu.java` class using any of the following constructors:
+
+```java
+//Using the API Key only
+String apiKey = "INSERT_API_KEY_HERE";
+// Using the API Key only
```
Comment (mayorJAY) [inline docs/docs/overview/quickstart/get-started-with-java.md:63]: You can remove lines 63 and 70

Comment (Basfar) [reply docs/docs/overview/quickstart/get-started-with-java.md]: Fixed

File: docs/docs/overview/quickstart/get-started-with-java.md (hunk 1 above)
Comment (Basfar) [reply docs/docs/overview/quickstart/get-started-with-java.md]: Fixed

File: docs/docs/overview/quickstart/get-started-with-java.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+        return null;
+```
+
+### Sending a Notification to a Topic
+
+Thanks to the topics feature, it is possible to trigger a notification to all subscribers assigned to a topic. This helps avoid listing all subscriber identifiers in the `to` field of the notification trigger.
+
+:::info
+This feature is not yet available on the Java SDK, this doc will be updated once it is released on Java.
+:::
```
Comment (mayorJAY) [inline docs/docs/overview/quickstart/get-started-with-java.md:324]: This feature is actually available on Java. Remember the `to` field can also accept a Topic or list of Topics

Comment (Basfar) [reply docs/docs/overview/quickstart/get-started-with-java.md:324]: Noted

Comment (Basfar) [reply docs/docs/overview/quickstart/get-started-with-java.md]: Resolved

**Review by unicodeveloper — APPROVED** [review]

**Comment by unicodeveloper** [comment]
I can go ahead and merge this, but this means it won't be available or live on the docs site until we tag another major release.

So is it possible for you to go ahead to the main branch, fetch the latest and submit a fresh PR against the main branch? @Basfar

**Pull request closed without merging**